ape/ape-m1.c

```diff
@@ -964,7 +988,20 @@ int main(int argc, char **argv, char **envp) {
   sp = sp2;
 
   /* interpret command line arguments */
-  if ((M->ps.literally = argc >= 3 && !StrCmp(argv[1], "-"))) {
+  if ((M->ps.indirect = GetIndirectOffset(argv[0]))) {
+    M->ps.literally = 0;
+    /* if argv[0] is $prog.ape, then we strip off the .ape and run
```

Comment:
Could you explain the use case a bit more?

> [Existing comment]. This lets you use symlinks to trick the OS at times when a native executable is required. For example, let's say you want to put the APE binary `/opt/cosmos/bin/bash` in your `/etc/shells`, or perhaps you want to use it as a shebang interpreter (i.e. `#!/opt/cosmos/bin/bash`). That won't work with APE normally, but it will if you say `ln -sf /usr/local/bin/ape /opt/cosmos/bin/bash.ape` and then use `#!/opt/cosmos/bin/bash.ape` instead.

Speaking of which, we might want to do this for `ape/loader.c` too.

Reply:
Filed #979 for `ape/loader.c`.